src/main/java/com/mattermost/integration/figma/notification/service/FileNotificationService.java

```diff
@@ -140,27 +145,41 @@ private UserDataDto getCurrentUserData(String userId, String mmSiteUrl, String b
                 botAccessToken), UserDataDto.class).get();
     }
 
-    private String createFileNotificationMessage(FigmaWebhookResponse figmaWebhookResponse) {
-        return String.format("User %s mentioned you in a comment %s in a file %s",
-                figmaWebhookResponse.getTriggeredBy().getHandle(),
-                prepareComment(figmaWebhookResponse.getComment(), figmaWebhookResponse.getMentions()),
-                figmaWebhookResponse.getFileName());
+    private String createFileNotificationMessage(FigmaWebhookResponse figmaWebhookResponse, String mmSiteUrl,
+                                                 String botToken) {
+        UserDataDto userDataDto = getCurrentUserData(figmaWebhookResponse.getTriggeredBy().getId(), mmSiteUrl, botToken);
+        List<MMUser> mmUsers = getMMUsersByIds(Collections.singletonList(userDataDto.getMmUserId()),
+                mmSiteUrl, botToken);
+        return String.format("%s posted a new comment in a file \"%s\":\n \"%s\" ",
+                "@".concat(mmUsers.get(0).getUsername()),
+                figmaWebhookResponse.getFileName(),
+                prepareComment(figmaWebhookResponse.getComment(), figmaWebhookResponse.getMentions(), mmSiteUrl, botToken));
     }
 
-    private String prepareComment(List<Comment> comments, List<Mention> mentions) {
+    private String prepareComment(List<Comment> comments, List<Mention> mentions, String mmSiteUrl,
+                                  String botToken) {
+        List<UserDataDto> userData = mentions.stream().map(mention ->
+                getCurrentUserData(mention.getId(), mmSiteUrl, botToken)).collect(Collectors.toList());
+        List<MMUser> mmUsers = getMMUsersByIds(userData.stream().map(UserDataDto::getMmUserId).collect(Collectors.toList()),
+                mmSiteUrl, botToken);
+
         StringBuilder stringBuilder = new StringBuilder();
         int mentionCounter = 0;
         for (Comment comment : comments) {
             if (Objects.nonNull(comment.getText())) {
                 stringBuilder.append(comment.getText());
             } else {
```

Comment:
if you want you can also replace here to a stream logic 
https://ivarconr.wordpress.com/2013/11/20/java-8-joining-strings-with-stream-api/